src/sass/base/_va.scss

```diff
@@ -196,9 +196,11 @@ hr {
   clear: both;
   margin: 0;
   padding: 0;
+  min-height: 147.19px;
```

Comment:
Reduces jankiness when the menus pop in, needs some testing in other browsers still.

Reply:
Yeah, curious to see how/whether things jump around unpredictably depending on the device, but sounds like you're looking into it.  If you need people to QA on various devices once the change is on staging, maybe put out a request on #vetsdotgov

Reply:
Should we create a sign up sheet for this? A markdown table in the description where folks can add their name and indicate if things look ok on a specific browser?

Reply:
So far, testing on BrowserStack, it seems ok. I think zoom levels might affect it, too, so I'm going to play around with that some.

Reply:
@cehsu I think we're good. I was able to test our supported browsers without any trouble. It's the same on every page, so there are not a lot of things to test.